### DISCUSSION TITLE
Tiny UI suggestion in Roles & Permissions

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15254] [Feature Requests]
Currently, to create a user, the wording provided is **Create Item**. I'm wondering if we should change it to **Create User**, which is just a bit more specific and closer to context.

<img width="1552" alt="Screen Shot 2022-08-24 at 10 44 49 AM" src="https://user-images.githubusercontent.com/62306630/186463046-6668d75e-9764-46a8-92a6-3c19e39b27ca.png">

**Comment by azrikahar** [comment]
That is a good point 👍

Technically we can't change it to `Create User` _yet_ (hence the move to discussion for now), since we will need to implement the same "Create New label" option that is present in repeaters, and make it available in the relational interfaces first, as suggested in #12209.

Once that is in place, we should be able to implement this `Create User` label using said option over here:

https://github.com/directus/directus/blob/f91247260a96045e28b0cf9785c4ec0cc61d4f16/api/src/database/system-data/fields/roles.yaml#L48-L55

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
